aiken-verifier/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "committee-bridge-contract-aiken"
```

Comment:
it has nothing to do with committee-bridge at this point. It should be generic halo2 verifier